lib/src/_index.dart

```diff
@@ -118,6 +120,8 @@ class IndexRef {
       scroll: scroll,
       highlight: highlight,
       trackTotalHits: trackTotalHits,
+      size: size,
```

Comment:
I'm not aware of this feature, but based on a quick search, shouldn't this come in pairs of `from` + `size`? (Also not sure what's the difference between this and offset/limit)...

Reply:
hum... yeah indeed, maybe that is unnecessary. To be honest, I thought I didn't add it last time and it come in pairs with the Client.
I can revert this part if you prefer?

Reply:
It looks like, but I'm not sure, that `limit` and `offset` was a thing in the past, but it is not anymore. Were you using `size` to set a maximum limit on the result set returned? In that case we can either repurpose the current `offset` and `limit` parameters, or introduce the new pair... wdyt?

Reply:
I use `size` to limit the number of hits returned indeed. I use it for aggregations too, because the `limit` argument doesn't exist for aggregations.
Looks like `size` is the same as `limit` and `from` the same as `offset`.
I use it to do a pagination system.

The documentation do like this: https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html

But as this package haven't `from`, so I use `offset`.

When I'm looking for `offset` and `limit`, I don't find a lot of things in the documentation about them 🤔
Looks like `size` and `from` are the new norm.

What do you think too?

Reply:
Let's keep this PR as-is, and I'll publish it after merging. I think in the long term we want to follow ElasticSearch's rename policy, but it is not advisable to break client that may be using older ES. I'll add a comment that we should eventually deprecate `offset` and `limit`.

Reply:
Yes I agree. Better deprecated it than remove it.